stratum/hal/lib/pi/pi_node.cc

```diff
@@ -35,9 +35,10 @@ ::util::Status toUtilStatus(const DeviceMgr::Status& from,
     if (results->size() != 0)
       return MAKE_ERROR(ERR_INTERNAL) << "Expected empty results vector.";
     results->resize(updates_size);
-    return ::util::Status();
+    return ::util::OkStatus();
   }
-  ::util::Status status(::util::Status::canonical_space(), from.code(), "");
+  ::util::Status status(::util::Status::canonical_space(), from.code(),
+                        from.message());
```

Comment:
```suggestion
  ::util::Status status(from.code(), from.message());
```

Reply:
Good idea, but the compiler disagrees:

```
stratum/hal/lib/pi/pi_node.cc:40:18: error: no matching constructor for initialization of '::util::Status'
  ::util::Status status(from.code(), from.message());
                 ^      ~~~~~~~~~~~~~~~~~~~~~~~~~~~
./stratum/glue/status/status.h:71:3: note: candidate constructor not viable: no known conversion from '::google::protobuf::int32' (aka 'int') to '::util::error::Code' for 1st argument
  Status(::util::error::Code code, const std::string& error_message);
  ^
```

Reply:
ah, I forgot that is the enum. the current code is good enough I think